dali/driver/uart.py

```diff
@@ -0,0 +1,109 @@
+
+"""
+UART communication driver for DALI
+"""
+
+import logging
+import serial
+import struct
+from dali.command import Command
+from dali.interface import DriverInterface, ParameterError
+
+
+class DaliUART(DriverInterface):
```

Comment:
It's sort-of self-made, but I can't release any part of it. I do believe that anyone can build a microcontroller stuff at home, and implement this. [Like this one.](http://siwawi.bauing.uni-kl.de/avr_projects/dali/)

I'll rename it to RAW, or something.